docs/PULL_REQUEST_TEMPLATE.md

```diff
@@ -6,31 +6,40 @@ Please include a summary of the change and which issue is fixed. Please also inc
 
 Please delete options that are not relevant.
 
-- [ ] Bug fix (non-breaking change which fixes an issue)
-- [ ] New feature (non-breaking change which adds functionality)
-- [ ] Documentation update
-- [ ] Code cleanliness or refactor
-- [ ] Dependency change
+- Bug fix (non-breaking change which fixes an issue)
+- New feature (non-breaking change which adds functionality)
+- Documentation update
+- Code cleanliness or refactor
+- Dependency change
```

Comment:
Instead of a checklist, these all felt like a list since you can just delete what's not relevant.